db/migrate/20190502190921_create_topics.rb

```diff
@@ -0,0 +1,9 @@
+class CreateTopics < ActiveRecord::Migration[5.2]
+  def change
+    create_table :topics do |t|
+      t.string :name
```

Comment:
Maybe it would be nice to have some validations on the DB layer, like unique & not null, wdyt?